basicswap/static/js/bids_sentreceived.js

```diff
@@ -226,13 +365,37 @@ const getStatusClass = (status) => {
         case 'Completed':
             return 'bg-green-300 text-black dark:bg-green-600 dark:text-white';
         case 'Expired':
+        case 'Timed-out':
             return 'bg-gray-200 text-black dark:bg-gray-400 dark:text-white';
         case 'Error':
-            return 'bg-red-300 text-black dark:bg-red-600 dark:text-white';
         case 'Failed':
+        case 'Failed, swiped':
             return 'bg-red-300 text-black dark:bg-red-600 dark:text-white';
         case 'Failed, refunded':
-            return 'bg-gray-200 text-black dark:bg-gray-400 dark:text-red-500';
+            return 'bg-red-300 text-black dark:bg-red-600 dark:text-white';
```

Comment:
```suggestion
            return 'bg-red-300 text-black dark:bg-red-600 dark:text-white';
         case 'Failed, refunded': 
         case 'Failed, swiped': 
            return 'bg-gray-200 text-black dark:bg-gray-400 dark:text-red-500';

```

Using different color for failed swaps that were refunded or swiped

maybe gray isn't the best color